### PR TITLE
Autos

### PR DESCRIPTION
**Pull request by rotatingcow**
Added path planning / following

### CODE REVIEW

**Review by Gavin-Niederman — APPROVED** [review]
LGTM (Lego Game That Moves)

File: src/main/java/com/frcteam3636/frc2024/RemoteADStarAK.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        }
+
+        @Override
+        public void fromLog(LogTable table) {
+            isNewPathAvailable = table.get("IsNewPathAvailable", false);
+
+            double[] pointsLogged = table.get("CurrentPathPoints", new double[0]);
+
+            List<PathPoint> pathPoints = new ArrayList<>();
+            for (int i = 0; i < pointsLogged.length; i += 2) {
```
Comment (rezetta15) [inline src/main/java/com/frcteam3636/frc2024/RemoteADStarAK.java:128]: 2? What is 2?

Comment (rotatingcow) [reply src/main/java/com/frcteam3636/frc2024/RemoteADStarAK.java:128]: cause when the points are added it adds x point and y point, so to go to the next logged point i += 2

**Review by rezetta15 — APPROVED** [review]
So magic numbers in code (other than obvious things like 0, 1, 60, 360) give me hives but that might be a me problem. But I pretty much understood, with Zac's help, what was going on. In general it should be considered by the team to explain and/or define what any explicit number means. 

You guys are amazing and should be very proud of the work you have done. Seriously.

File: src/main/java/com/frcteam3636/frc2024/subsystems/drivetrain/Drivetrain.kt [hunk 2]
```diff
@@ -160,21 +194,27 @@ object Drivetrain : Subsystem {
...
 
             val chassisSpeeds = ChassisSpeeds(
-                translationJoystick.x * FREE_SPEED,
-                translationJoystick.y * FREE_SPEED,
+                FREE_SPEED.baseUnitMagnitude() * translationJoystick.x,
+                FREE_SPEED.baseUnitMagnitude() * translationJoystick.y,
                 0.0
             )
+
+            TODO()
```
Comment (doinkythederp) [inline src/main/java/com/frcteam3636/frc2024/subsystems/drivetrain/Drivetrain.kt:210]: This needs to be finished or the binding for it needs to be removed.

**Review by doinkythederp — CHANGES_REQUESTED** [review]

**Review by doinkythederp — APPROVED** [review]

**Review by max-niederman — APPROVED** [review]
LGTM

**Pull request merged**